.github/workflows/label-pr.yml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This workflow utilizes https://github.com/actions/github-script in GitHub Actions to apply labels to pull requests.
+#
+
+name: Label PR
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Label PR
+        uses: actions/github-script@v4
+        with:
+          async: yes
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What happens if you use the `AGONES_BOT` secret?

Reply:
I attempted to use AGONES_BOT by creating a secret with that name in my repo, but it's not functioning as intended.- https://github.com/Kalaiselvi84/agones/actions/runs/5133792952/jobs/9236886161?pr=262

Reply:
Let's take that for a spin, see how it goes.

Reply:
I created a new PR to test this script, but it did not receive any labels. Instead, I encountered an error message stating, "Error retrieving files: HttpError: Resource not accessible by integration." - https://github.com/googleforgames/agones/actions/runs/5140174858/jobs/9251379449?pr=3194

Reply:
That's weird. So it worked once but not a second time?

Should we just merge it and see what happens on the Agones repo? Any downside?